notebooks/user_guide.ipynb

```diff
@@ -3553,7 +3553,9 @@
     "\n",
     "\"Quantifies the probability that two interacting pedestrians are found a given distance r apart, renormalized by the probability $P_{Ni}$ of measuring this distance for pedestrians that do not interact.\"\n",
     "\n",
-    "This probability is approximated here by randomising the time information (randomisation of the time frames).\n",
+    "In this method, \"interacting pedestrians\" are defined as pedestrians that are present in the same spatial domain at the same time. One should also keep in mind that in its current implementation, the method does not take into account walls and corners, which should in theory block any \"interaction\" between pedestrians on opposite sides of the obstacles.\n",
+    "\n",
+    "The probability $P_{Ni}$ is approximated here by time randomising the original trajectory file. For this randomisation process, only the frame numbers of the trajectory file are shuffled. The created \"randomised trajectories\" contains random pedestrian positions, composed only of positions which were present in the original trajectory file. This method helps to take into account preferred route choices of participants, which can be due to terrain features or social behaviours.\n",
```

Comment:
I don't understand the last sentence and I' m not sure it it is necessary to talk about routing in this context.